src/styles/blocks/button.scss

```
@@ -0,0 +1,39 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  height: 56px;
+  border: 2px solid $colorBlue;
+  border-radius: 8px;
+
+  font-size: 16px;
+  font-family: inherit;
+  font-weight: bold;
+  line-height: 14px;
+  text-decoration: none;
+  color: $colorBlue;
+
+  cursor: pointer;
+  transition: all 0.3s;
```

Always add a transition for styles that you want smoothly change 
And create transition-duration variable 

```suggestion
  transition: background-color $transition-duration, border-color 0.3s, color 0.3s;
```